openid-4-verifiable-presentations-1_0.md

```diff
@@ -2764,57 +2576,29 @@ be valid type identifiers as defined in [@!I-D.ietf-oauth-sd-jwt-vc]. The Wallet
 the inheritance logic defined in [@!I-D.ietf-oauth-sd-jwt-vc].
 
-#### SD-JWT VC DCQL Query example
+### Presentation Response
 
 A non-normative example DCQL query using the SD-JWT VC format is shown in (#dcql_query_example).
 The respective response is shown in (#response_dcql_query).
 
 Additional examples are shown in (#more_dcql_query_examples).
 
-### Presentation Request
-
-The following is a non-normative example of an Authorization Request:
-
-<{{examples/request/request.txt}}
-
-The following is a non-normative example of the contents of a `presentation_definition` parameter that contains the requirements regarding the Credential to be presented:
-
-<{{examples/request/pd_sd_jwt_vc.json}}
-
-The presentation of an IETF SD-JWT VC is requested by adding an object named `dc+sd-jwt` to the `format` object of an `input_descriptor`. The `input_descriptor` value is applied to the unsecured payload of the IETF SD-JWT VC which correspond to the disclosures of the presented SD-JWT VC.
-
-Setting `limit_disclosure` property defined in [@!DIF.PresentationExchange] to `required` enables selective release by instructing the Wallet to submit only the disclosures for the matching claims specified in the fields array. The unsecured payload of an IETF SD-JWT VC is used to perform the matching.
-
-### Presentation Response
-
-A non-normative example of the Authorization Response would look the same as in the examples of other Credential formats in this Annex.
-
 The following requirements apply to the `nonce` and `aud` claims in the Key Binding JWT:
 
 - the `nonce` claim MUST be the value of `nonce` from the Authorization Request;
 - the `aud` claim MUST be the value of the Client Identifier, except for requests over the DC API where it MUST be the Origin prefixed with `origin:`, as described in (#dc_api_response).
 
-The `transaction_data_hashes` response parameter defined in (#transaction_data) MUST be included in the Key Binding JWT as a top level claim. This means that transaction data mechanism cannot be used with SD-JWT VCs without cryptographic key binding and, therefore, do not use KB JWT.
-
-The following is a non-normative example of the content of the `presentation_submission` parameter:
-
-<{{examples/response/ps_sd_jwt_vc.json}}
-
-The following is a non-normative example of the `vp_token` parameter provided in the same response and referred to by the `presentation_submission` above:
-
-<{{examples/response/token_response_vp_token_sd_jwt_vc.txt}}
-
-In this example the `vp_token` contains only the disclosures for the claims specified in the `presentation_submission`, along with a Key Binding JWT.
+The `transaction_data_hashes` response parameter defined in (#transaction_data), if Transaction Data is used, MUST be included in the Key Binding JWT as a top level claim. This means that transaction data mechanism cannot be used with SD-JWT VCs without cryptographic key binding (i.e., which do not use the KB-JWT).
```

Comment:
this will probably conflict with #421. tho #421 probably getting merged first. @sander